price/pkg/settings-template.toml

```diff
@@ -0,0 +1,56 @@
+# log settings for the application (RUST_LOG format). Default below
+# 
+# log = "price=debug"
+
+# RPC Endpoint for price oracles. Required.
+rpc_endpoint = "https://api.devnet.solana.com"
```

Comment:
Rename to source. 

In general config keys with a `_` in it mess with the ability to override them in env variables

Reply:
This will need to be overridden to one of the better endpoints during deployment

Reply:
Fixed in [fe83739](https://github.com/helium/oracles/pull/400/commits/fe837396d363ec7094be174464b50e8b89359488)